docs/package_info/contributing.rst

```diff
@@ -105,7 +104,8 @@ Static Checker
 
 PRAW's test suite comes with a checker tool that can warn you of using incorrect
 documentation styles (using ``.. code-block::`` instead of ``.. code::``, using ``/r/``
```

Comment:
Noticed this while reading the paragraph. 
```suggestion
documentation styles (using ``.. code::`` instead of ``.. code-block::``, using ``/r/``
```
These should be switched, would you mind fixing that with this PR?

Reply:
I can, but are you sure about that? The existing docs, including this very contributing guide itself, all use `.. code-block::`, and various [sources](https://stackoverflow.com/questions/34845889/whats-the-difference-between-the-code-and-code-block-directives-in-rest) and [style](https://github.com/spyder-ide/spyder-docs/blob/master/STYLEGUIDE.md#rest-style-elements) [guides](https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/WritingReST/Codeblocks.html) I looked at (full disclosure: I wrote the Spyder one) all recommend using `code-block` over `code` when using Sphinx due to the latter being a modern Sphinx directive with more sophisticated options, syntax highlighting and more.

Reply:
I am. This is outlining two examples that are incorrect. The way I read this sentence is that the latter is the preferred way over the former.

Reply:
Ah, you're right. I should have checked the context, my mistake. Sorry for the delay, I was on standby for the SpaceX Inspiration4 mission in case of contingencies.

Reply:
Done, thanks.